docs/backends/macos.rst

```diff
@@ -28,7 +28,31 @@ As stated above, this will however only work the macOS machine that performed
 the scan and thus cached the device as ``243E23AE-4A99-406C-B317-18F1BD7B4CBE``.
 
 There is also no pairing functionality implemented in macOS right now, since it does not seem
-to be any explicit pairing methods in the COre Bluetooth.
+to be any explicit pairing methods in the Core Bluetooth.
+
+CoreBluetooth does not differentiate between data from a notification, and data from a read.
+This can cause confusion in cases where a device may send a notification message on a characteristic
+that is an indication that the characteristic needs to be read again.
```

Comment:
"indication" probably isn't the best choice of words here since it is the technical name of a BLE notification that requires a response. :wink:

Reply:
Great point!  That would be a bit confusing.  Reworded.